sdk/formrecognizer/Azure.AI.FormRecognizer/src/Constants.cs

```diff
@@ -1,16 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+using System;
 
 namespace Azure.AI.FormRecognizer
 {
     internal static class Constants
     {
+        private const string AzurePublicCloud = "https://login.microsoftonline.com/";
+        private const string AzureChina = "https://login.microsoftonline.cn/";
+        private const string AzureGovernment = "https://login.microsoftonline.us/";
+
         public const string AuthorizationHeader = "Ocp-Apim-Subscription-Key";
 
         public const string OperationLocationHeader = "Operation-Location";
 
-        public const string DefaultCognitiveScope = "https://cognitiveservices.azure.com/.default";
+        public static string DefaultCognitiveScope = getDefaultCognitiveScope(Environment.GetEnvironmentVariable("AZURE_AUTHORITY_HOST"));
```

Comment:
do you know where the users can get this value from? it will be good to update the documentation to tell the user that if they need to use another cloud, they need to set this variable.

FYI @schaabs for input on the approach as I am not sure it is what we recommend

Reply:
This value was defaulted in the arm template, and overridden in the live test config. Users don't need to set this variable.[https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L492](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L492)

Reply:
Sure, but a user using a client library doesn't know about our arm template so where do they get this value from?
Once this code is merged, user's existing code is going to break because their environment doesn't have the `AZURE_AUTHORITY_HOST` variable set.

If you really want to go with this approach and later on work on a way that is more user friendly, my recommendation is to always have a default value for `DefaultCognitiveScope ` and don't throw/error in case the env variable doesn't exist

Reply:
I have set a default value for [DefaultCognitiveScope](https://github.com/zedy-wj/azure-sdk-for-net/blob/99bc0f3e659c2a6c96926c9a23559170e3906b82/sdk/formrecognizer/Azure.AI.FormRecognizer/src/Constants.cs#L31), so it won't throw/error.

The value of `AZURE_AUTHORITY_HOST` in different clouds as follow:
Public : `https://login.microsoftonline.com/`
UsGov : `https://login.microsoftonline.us/`
China : `https://login.microsoftonline.cn/`